src/components/Header.vue

```diff
@@ -70,7 +104,10 @@ let testing = false;
 export default {
   name: 'Header',
   data() {
-    return { activityViews: [] };
+    return {
+      activityViewsDaily: [],
+      activityViewsSummary: [],
```

Comment:
Because of view.pathUrl

Reply:
I don't see how that changes anything, could you elaborate?

Reply:
So android devices can't go to a broken summaries page.

Reply:
That doesn't make sense. That's exactly what the `if(testing || types.android)` and `if(types.afk && types.window)` checks are for.

Reply:
How doesn't it make sense? pathUrl are different for android and desktop and I have not implemented summaries for android.

Reply:
I have read the code (several times by now), and well I'm aware you haven't implemented it on Android, but there's still no need to split up the activityViews list into two arrays. That the `pathUrl`s are different doesn't imply you need two arrays.

The code would work just fine with a single array, it would not show up on Android (because of the `types.android` check).

Reply:
You might be confused because you didn't correctly refactor some code when you moved stuff into the Header component (including not moving the isAndroidApp property), which was fixed here: https://github.com/ActivityWatch/aw-webui/pull/140/

Reply:
As long as pathUrl exists it's literally impossible to have it on two seperate buttons. If you want it refactored somehow say that but as it is now it is not possible to simply merge the lists.

Reply:
I'm sorry, I have not been clear: Why have two separate buttons? 

One dropdown should be enough? Would also make the left side of the header overflow on "small" lg-sized screens, so it breaks styling on such screens. There's also good breakpoint between md and lg we can use, so not an option unless we want to override Bootstrap default breakpoints (which we really shouldn't).

Reply:
I decided to have two buttons because having a dropdown is bad UX and as most people won't have multiple devices but all will have both daily and summary views. In the future I am hoping that we can merge "Daily" into a tab inside the summary view just as for "Week", "Month" and "Year" but that is currently not possible because:

- The daily view is not modular and split into suitable components
- The daily view contains event timelines which generate way too much data to be able to be shown within the summary views, either we need edge-cases for this within the summary view or we need to come up with something better.

Reply:
In that case, just merge it for now then. I'll probably modify it in the future.